roles/logstash/tasks/main.yml

```
@@ -11,6 +11,16 @@
   changed_when: false
   when: ansible_os_family == 'Debian'
 
+- name: Install Python Packages
+  ansible.builtin.package:
+    name: ["python3", "python3-pip"]
```

```suggestion
    name: 
      - python3
      - python3-pip
```
Matches our other code. And it's easier readable as well as easier maintainable.

Are you sure, this will work on all supported operating systems? I thought, package names are different.

---

Well, all checks have passed. 
If we have checks for all supported systems it should be fine. If not, we will have to build checks for the missing one.